test/powershell/Modules/Microsoft.PowerShell.Diagnostics/New-WinEvent.Tests.ps1

```diff
@@ -0,0 +1,33 @@
+Describe 'New-WinEvent' -Tags "CI" {
+
+    Context "New-WinEvent tests" {
+        
+        It 'Simple New-WinEvent' -Skip:(-not $IsWindows) {
```

Comment:
Yup, I saw that approach; but don't really like it; I think having skip conditions explicitly on each test makes it easier to investigate test failures.